syntax = "proto3";

package AcFunDanmu.Im.Message;

import "ChatSession.proto";

message SessionCreateResponse {
	ChatSession session = 1;
}
